source/features/github-bugs.css

```diff
@@ -77,3 +77,19 @@
 .pagehead-actions :is(.btn, summary) .octicon {
 	margin-right: 4px !important;
 }
+
+/* Avoid excessive repo title wrap by deprioritizing the repo header actions #5398 */
+:root .pagehead-actions {
+	flex-shrink: 3 !important;
+	display: inline-flex !important;
+	flex-wrap: wrap;
+	justify-content: flex-end;
+}
+
+:root .pagehead-actions > li {
+	margin: 0;
+}
+
+.pagehead-actions > li > * {
+	margin: 0 8px 8px 0;
+}
```

Comment:
Optional but I think using `gap` is a better solution, the intent is clearer and it doesn't create unwanted padding inside the wrapper.

Reply:
Gap is incompatible with the markup, refer to my review

Reply:
> Gap is incompatible with the markup, refer to my review

Seems to me like it's perfectly compatible? If you're talking about it not working in Chrome, it was just a matter of increasing the selector's specificity: https://github.com/refined-github/refined-github/pull/5622#issuecomment-1120209012

Reply:
## Your current PR

<img width="897" alt="Screen Shot 13" src="https://user-images.githubusercontent.com/1402241/167282212-81400c5f-8c59-48f9-b8c4-2e5350b68381.png">

## My PR

<img width="889" alt="Screen Shot 14" src="https://user-images.githubusercontent.com/1402241/167282216-f5b48c75-14a6-4310-9273-c6d7b79ce3bd.png">


---

This is due to this empty element:

<img width="336" alt="Screen Shot 15" src="https://user-images.githubusercontent.com/1402241/167282244-af0fad65-bc86-48ff-8759-678dca87baa8.png">

```html
<li>
  <div class="float-left" data-test-selector="pin-repo-button">
    <form
      data-turbo="false"
      action="/fregante/webext-dynamic-content-scripts/profile_pin"
      accept-charset="UTF-8"
      method="post"
    >
      <input
        type="hidden"
        name="authenticity_token"
        autocomplete="off"
      />
      <button
        title="No pin slots remaining in your profile"
        disabled="disabled"
        type="submit"
        data-view-component="true"
        class="btn-sm btn"
      >
        <svg
          aria-hidden="true"
          height="16"
          viewBox="0 0 16 16"
          version="1.1"
          width="16"
          data-view-component="true"
          class="octicon octicon-pin mr-2"
        >
          <path
            fill-rule="evenodd"
            d="M4.456.734a1.75 1.75 0 012.826.504l.613 1.327a3.081 3.081 0 002.084 1.707l2.454.584c1.332.317 1.8 1.972.832 2.94L11.06 10l3.72 3.72a.75.75 0 11-1.061 1.06L10 11.06l-2.204 2.205c-.968.968-2.623.5-2.94-.832l-.584-2.454a3.081 3.081 0 00-1.707-2.084l-1.327-.613a1.75 1.75 0 01-.504-2.826L4.456.734zM5.92 1.866a.25.25 0 00-.404-.072L1.794 5.516a.25.25 0 00.072.404l1.328.613A4.582 4.582 0 015.73 9.63l.584 2.454a.25.25 0 00.42.12l5.47-5.47a.25.25 0 00-.12-.42L9.63 5.73a4.581 4.581 0 01-3.098-2.537L5.92 1.866z"
          ></path></svg
        >Pin
      </button>
    </form>
  </div>
</li>


```

Reply:
Wait, I thought the "Pin"/"Unpin" button was replaced with a dropdown everywhere? But I guess it's only on organization repos then.

That means #5612 needs to be amended.